dataservice/extensions/flask_indexd.py

```diff
@@ -35,6 +35,52 @@ def teardown(self, exception):
         if hasattr(ctx, 'indexd_session'):
             ctx.indexd_session.close()
 
+    def merge_properties(self, uuid, record):
```

Comment:
Having this be a method of the indexd client doesn't make much sense to me. If you're going to mutate the state of an object's properties, it would seem more appropriate to have the object be responsible for doing it.

Reply:
Could we maybe break down the flask_indexd module and put the methods (merge_properties, register_record, update_record, delete_record) that really modify GenomicFile in the resource class? That class already seems to modify GenomicFile (i.e. in update method). I left them in flask_indexd because I saw the other methods already access the GenomicFile object quite a bit anyway.